### DISCUSSION TITLE
How to enable and disable plugins

### DISCUSSION THREAD

**Discussion by Hex-Tron** [LunarVim/LunarVim#4609] [Q&A]
Hey guys, Hope U doing great.
I am facing difficulty in enabling and disabling plugins please provide me some info

![image](https://github.com/user-attachments/assets/95ca729b-19bd-4892-ae7e-fc6ee604baad)

---
I have also tried discord seems its verification is not working properly please consider that too.
![image](https://github.com/user-attachments/assets/ae4d1d21-8817-4889-be3e-4af119ebfa19)

**Comment by juicy-g** [comment]
In your top screenshot, the nvim-cmp plugin is installed and enabled. It's just not loaded yet. In Lunarvim the nvim-cmp plugin is lazy-loaded after the `InsertEnter` or `CmdlineEnter` events. 

If you are looking to disable a builtin plugin, like nvim-cmp for example, then you can do `lvim.builtin.cmp.active = false`.
